src/main/query_profiler.cpp

```diff
@@ -454,6 +442,71 @@ string QueryProfiler::QueryTreeToString() const {
 	return str.str();
 }
 
+string RemovePhaseTimingName(string name, string metric) {
+    return metric.substr(name.size());
+}
+
+void RenderPhaseTimings(std::ostream &ss, pair<string, double> head, map<string, double> timings, idx_t width) {
+	ss << "┌────────────────────────────────────────────────┐\n";
+	ss << "│" +
+	         QueryProfiler::DrawPadded(RenderTitleCase(head.first) + ": " + RenderTiming(head.second),
+	                     width - 2) +
+	          "│\n";
+	ss << "│┌──────────────────────────────────────────────┐│\n";
+
+	for (const auto &entry : timings) {
+			ss << "││" +
+		          QueryProfiler::DrawPadded(RenderTitleCase(entry.first) + ": " + RenderTiming(entry.second),
+			                     width - 4) +
+			          "││\n";
+    }
+    ss << "│└──────────────────────────────────────────────┘│\n";
+    ss << "└────────────────────────────────────────────────┘\n";
+}
+
+void PrintPhaseTimingsToStream(std::ostream &ss, const ProfilingInfo info, idx_t width) {
+	map<string, double> optimizer_timings;
+	map<string, double> planner_timings;
+	map<string, double> physical_planner_timings;
+
+	pair<string, double> optimizer_head;
+	pair<string, double> planner_head;
+	pair<string, double> physical_planner_head;
```

Comment:
This is for backward compatibility with the printing?

Reply:
Yes exactly.